internal/install/application_configuration_yml.go

```diff
@@ -5,10 +5,11 @@
 package install
 
 const (
-	elasticAgentImageName         = "docker.elastic.co/beats/elastic-agent"
-	elasticAgentCompleteImageName = "docker.elastic.co/beats/elastic-agent-complete"
-	elasticsearchImageName        = "docker.elastic.co/elasticsearch/elasticsearch"
-	kibanaImageName               = "docker.elastic.co/kibana/kibana"
+	elasticAgentImageName               = "docker.elastic.co/beats/elastic-agent"
+	elasticAgentCompleteLegacyImageName = "docker.elastic.co/beats/elastic-agent-complete"
+	elasticAgentCompleteImageName       = "docker.elastic.co/elastic-agent/elastic-agent-complete"
```

Comment:
@jlind23 I understand that it also applies to `elastic-agent-complete` as we're using the full image version.